src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
Yeah... I too have thought about something like that; I think that maybe we can do something better later on—for now, I think we can leave the `_` in, but ultimately there are some alternatives that may turn out to be preferable in the end. Let's keep thinking about this, but put it on the back-burner.